pkg/connect/client/trustzone/v1alpha1/trustzone.go

```diff
@@ -48,7 +48,7 @@ func (c *trustZoneClient) CreateTrustZone(ctx context.Context, trustZone *trustz
 }
 
 func (c *trustZoneClient) GetTrustZone(ctx context.Context, trustZoneID string) (*trustzonepb.TrustZone, error) {
-	resp, err := c.trustZoneClient.GetTrustZoneDetails(ctx, &trustzonesvcpb.GetTrustZoneDetailsRequest{
```

Comment:
We should handle backward compatibility here. We can try calling GetTrustZone first, then check for an error and convert to a gRPC status (status.Convert) then check the code for codes.Unimplemented.